src/pylint_datetime/__init__.py

```diff
@@ -82,7 +82,7 @@ def visit_call(self, node: nodes.Call) -> None:
         else:
             assert False, "function call without name or attrname"
 
-        if func_name == "timedelta":
+        if func_name in ("timedelta", "datetime"):
             for argument in node.args:
                 if argument.value != 0:
                     self.add_message("datetime-timedelta-no-keyword-args", node=node)
```

Comment:
Note that `datetime` shouldn't be allowed as `datetime()` not `datetime(0)` like `timedelta`, so you should account for this difference.

I think we should forbid this also for `date` (also forbidding `date()` and `date(0)`).

`time` should be allowed, as I don't think anyone really ever uses seconds, minutes, hours, it is always hours, minutes, seconds.